app/routes/circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -9,6 +9,7 @@ import { Form, Link, useNavigation } from '@remix-run/react'
 import {
   Button,
   ButtonGroup,
+  DatePicker,
```

Comment:
I think that we record the time with second precision. We probably need a date-time picker.

Reply:
How accurate do we care about making it? I may need to revise this a bit, as I assumed the only time this would be used is if the date is not defined, like on https://gcn.nasa.gov/circulars/12, but I am now seeing it DOES have a time (just 0).

I also assumed we did not know the exact hour/min/sec time that one of these older circulars came in, is that true?

Reply:
In many cases we know down to the hour or even more accurately when these occurred because these were intensely studied events. The time may be apparent from the text of the Circular. There was even a book written for lay people about GRBs that probably has a pretty accurate timeline for some of those early GRBs.

Who knows, the timestamp on the filesystem on capella2 may even be accurate.

Are we sure that we want to disable date corrections for Circulars that already have dates? That makes this kind of a one-off feature.

Reply:
Hm thats a good point. I didn't know how much data was tracked on the initial files that came in, but if we know with that degree of accuracy we should preserve it.

 I am not opposed to making it enabled for any circular. Its entirely likely that some newer ones may need to be updated. 

The react uswds library does not contain a Datetime picker, just distinct Date and Time pickers. But I could probably whip  one up and commit it to their repo when it is ready

Reply:
I think a date picker plus a time picker would be fine.